packages/devkit/src/mathml.js

```diff
@@ -276,35 +276,37 @@ export default class MathML {
 
   /**
    * Removes semantics tag to mathml.
+   * When using Hand to create formulas, it adds the mrow tag due to the semantics one, this one is also removed.
    * @param {string} mathml - MathML string.
    * @returns {string} - 'mathml' without semantics tag.
    */
   static removeSemantics(mathml) {
     const mathTagEnd = '</math>';
     const openSemantics = '<semantics>';
-    const openAnnotation = '<annotation';
+    const openAnnotationRegex = /<annotation(.*)/g;
+    const openSemanticsMrow = '<semantics><mrow>';
+    const openAnnotationMrowRegex = /<\/mrow><annotation(.*)/g;
 
     let mathmlWithoutSemantics = mathml;
-    const startSemantics = mathml.indexOf(openSemantics);
-    if (startSemantics !== -1) {
-      const startAnnotation = mathml.indexOf(openAnnotation, startSemantics + openSemantics.length);
-      if (startAnnotation !== -1) {
-        mathmlWithoutSemantics = mathml.substring(0, startSemantics)
-        + mathml.substring(startSemantics + openSemantics.length, startAnnotation) + mathTagEnd;
-      }
-    }
+
+    // Remove the semantics and mrow
+    mathmlWithoutSemantics = mathmlWithoutSemantics.replaceAll(openSemanticsMrow, '');
+    mathmlWithoutSemantics = mathmlWithoutSemantics.replaceAll(openSemantics, '');
+    mathmlWithoutSemantics = mathmlWithoutSemantics.replaceAll(openAnnotationMrowRegex, ''); // Regex to remove all after the mrow + annotation tag.
+    mathmlWithoutSemantics = mathmlWithoutSemantics.replaceAll(openAnnotationRegex, ''); // Regex to remove all after the annotation tag.
+    mathmlWithoutSemantics += mathTagEnd;
```

Comment:
Me parece increíble que realmente no hiciera falta ninguna de los ifs que había antes, hasta el punto que me hace sospechar :joy:.

Por cierto, creo que incluso se podría mejorar un poco más si este código fuera: 

```suggestion
return mathml.replaceAll(openSemanticsMrow, '')
    .replaceAll(openSemantics, '')
    .replaceAll(openAnnotationMrowRegex, '')
    .replaceAll(openAnnotationRegex, '')
    .concat(mathTagEnd);
```

(las lineas 290 y 299 también sobrarían)